src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -589,14 +589,22 @@
         //   "choice": "azure",
         //   "description": "Azure Pipelines",
         //   "displayName": "Azure Pipelines"
-        // },
-        // {
-        //   "choice": "github",
-        //   "description": "GitHub Actions",
-        //   "displayName": "GitHub Actions"
+        },
+        {
+          "choice": "github",
+          "description": "GitHub Actions",
+          "displayName": "GitHub Actions"
         }
       ]
     },
+    "defaultBranchName": {
+      "displayName": "Default Branch Name",
+      "datatype": "text",
+      "description": "The name of the default branch for the repository",
+      "type": "parameter",
+      "defaultValue": "main",
+      "replaces": "$DefaultBranchName$"
```

Comment:
Let's remove this parameter - it's getting way too complex and just adding another value we need to expose in the wizard. We aren't going to expose "release" as a variable, so let's not expose main/master either